onnxscript/evaluator.py

```diff
@@ -380,28 +381,22 @@ def _ort_to_os_value(v):
     """Converts an ORT encoding of an ONNX value into the encoding used by onnxscript."""
     if isinstance(v, np.ndarray):
         return tensor.Tensor(v)
+    if np.issctype(type(v)):
+        # E.g. np.int64
```

Comment:
numpy scalar?

Reply:
They are numpy scalars. Interestingly they are different from arrays. Are you suggesting to update the comments?

Reply:
Yes that would be helpful, thanks.

Reply:
Done